### PR TITLE
[READY] Add download syllabus form

### PR DESCRIPTION
**Pull request by db0sch**
- Add a button on program page
- Modal with form (email, name, city)
- Save prospect with `origin` flag : 'syllabus'
- handle errors (if email already used)
- Display a nice letter opening when Prospect has been saved

TODO:
- [x] write some tests
- [x] Style the dropdown (default style at the moment)
- [x] it doesn't actually send an email. waiting for customer.io integration
- [x] i18n
- [x] get the city from the session

screenshot-2018-3-15 program - 9 weeks to learn ruby on rails
screenshot-2018-3-15 program - 9 weeks to learn ruby on rails 1
screenshot-2018-3-15 program - 9 weeks to learn ruby on rails 2
screenshot-2018-3-15 program - 9 weeks to learn ruby on rails

### CODE REVIEW

**Comment by db0sch** [comment]
just added the tests

**Comment by db0sch** [comment]
the dropdown arrow has been customized


screenshot 2018-03-15 17 34 19

screenshot 2018-03-15 17 34 34

**Comment by ssaunier** [comment]
@sandrineay the back-end is now ready! We can plug it to a CRM 👌

**Comment by db0sch** [comment]
i18n: 
- [ ] add fr (and other) translation to Hiring modal
- [ ] add keys in fr and en for Syllabus modal

File: config/environments/development.rb [hunk 1]
```diff
@@ -21,9 +21,10 @@
 
   # Don't care if the mailer can't send.
   config.action_mailer.raise_delivery_errors = true
-  # config.action_mailer.delivery_method = :letter_opener
+  config.action_mailer.delivery_method = :letter_opener
```
Comment (db0sch) [inline config/environments/development.rb]: du coup, pour tester en local, je voulais ouvrir les emails avec letter opener.
mais je peux remettre en smtp, si tu preferes.

**Comment by ssaunier** [comment]
👏

**Pull request merged**